rdrf/rdrf/templates/registration/password_change_form.html

```diff
@@ -0,0 +1,37 @@
+{% extends "rdrf_cdes/base.html" %}
+{% load i18n %}
+{% load static from staticfiles %}
+
+{% block content %}
+
+<div class="row">
+    <div class="col-md-6 col-md-offset-3">
+        <div class="alert alert-info">
+            {% trans "Please enter your old password, for security's sake, and then enter your new password twice so we can verify you typed it in correctly." %}
```

Comment:
Edit to text:

"Please enter your old password, and then enter your new password twice so we can verify you typed it in correctly."